dev-docs/modules/oneKeyRtdProvider.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: page_v2
+title: OneKey RTD Module
+display_name: OneKey RTD Module
+description: OneKey Real Time Data Module
+page_type: module
+module_type: rtd
+module_code : oneKeyRtdProvider
+enable_download : false
+vendor_specific: true
+sidebarType : 1
+---
+
+# OneKey Data Provider
+
+{:.no_toc}
+
+* TOC
+{:toc}
+
+## Overview
+
+The OneKey real-time data module in Prebid has been built so that publishers
+can quickly and easily setup the OneKey Addressability Framework.
+This module is used along with the oneKeyIdSystem to pass OneKey data to your partners.
+Both modules are required. This module will pass transmission requests to your partners
+while the oneKeyIdSystem will pass the oneKeyData.
+
+Background information:
+- [prebid/addressability-framework](https://github.com/prebid/addressability-framework)
```

Comment:
We cannot merge until this reference is remove

Reply:
Fixed.